pkg/appimport/appimport_test.go

```diff
@@ -13,69 +13,37 @@ import (
 
 	"github.com/drud/ddev/pkg/testcommon"
 	"github.com/drud/ddev/pkg/util"
-	"github.com/drud/ddev/pkg/version"
-	"github.com/drud/drud-go/utils/dockerutil"
-	"github.com/drud/drud-go/utils/system"
 	homedir "github.com/mitchellh/go-homedir"
 	"github.com/stretchr/testify/assert"
 )
 
 const netName = "ddev_default"
 
 var (
-	testArchiveURL  = "https://github.com/drud/wordpress/releases/download/v0.1.0/db.tar.gz"
 	testArchivePath = path.Join(testcommon.CreateTmpDir("appimport"), "db.tar.gz")
-	composePath     = path.Join("testing", "db-compose.yaml")
 	cwd             string
 )
 
 func TestMain(m *testing.M) {
-	err := system.DownloadFile(testArchivePath, testArchiveURL)
+	testFile, err := os.Create(testArchivePath)
```

Comment:
Well, how about refactoring it into util.TouchFile()? That way it would be obvious what was going on, and usable if we need it again, or replaceable with a simpler golang technique when we discover one.